pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUserStudyEnrollment.java

```diff
@@ -307,6 +310,18 @@ default long changeUserStudyEnrollmentStatus(long userId,
         return changeUserStudyEnrollmentStatus(userId, studyId, newEnrollmentStatus, null);
     }
 
+    default long suspendUserStudyConsent(String userGuid, String studyGuid) {
+        //update EnrollmentStatus to EnrollmentStatusType.CONSENT_SUSPENDED
+        //Update All existing activity instances as read-only
+        long id = changeUserStudyEnrollmentStatus(
+                userGuid, studyGuid, EnrollmentStatusType.CONSENT_SUSPENDED, null);
+        Set<Long> instanceIds = getActivityInstanceDao().findActivityInstanceIdsByStudyAndUser(studyGuid, userGuid);
+        long userId = getJdbiUser().getUserIdByGuid(userGuid);
```

Comment:
We have `policy.getStudyId()` and `candidate.getParticipantUserId()`. I think using ids instead of guids will simplify these changes.

Reply:
Its easy to use Ids but felt guid's is more user / client friendly way.
Will change to Ids